README.md

```diff
@@ -108,11 +109,11 @@ The quick start assumes you are in a bash shell.
 
 ### Using `coinbrew`
 
-To build CoinUtils from source, obtain the `coinbrew` script, do
+To build Osi from source, obtain the `coinbrew` script, do
 ```
 wget https://raw.githubusercontent.com/coin-or/coinbrew/master/coinbrew
 chmod u+x coinbrew
-./coinbrew fetch Osi@master
+./coinbrew fetch Osi@gh-actions
```

Comment:
This is going to be merged into master, no? I think all the references to gh-actions in the readme should be reverted.

Reply:
The README is auto-generated and the correct branch is filled in automatically. Since this was generated in the `gh-actions` branch, this is correct. It will just need to be re-generated after the merge. The URLs for CI are probably also incorrect right now.